build.js

```diff
@@ -224,6 +292,7 @@ const bundle = () => {
 };
 
 async function main() {
+  emojiMapURL = await loadEmoji();
```

Comment:
consider to remove global var `emojiMapURL` by injecting into processing function(s)

Reply:
Well, it's true that global variable are considered a bad practice. But this is to be considered as a function that just happened to be a HashMap. That's why I used it that way.

Reply:
That's fine.